### PR TITLE
Adds content_counts to Namespace

### PR DESCRIPTION
**Pull request by chouseknecht**
For each namespace, show a count of the related content items by type.

```
"content_counts": {
    "action_plugin": 1,
    "module_utils": 6,
    "lookup_plugin": 3,
    "module": 8,
    "role": 6,
    "filter_plugin": 2,
    "strategy_plugin": 3
},
```

### CODE REVIEW

**Comment by chouseknecht** [comment]
@cutwater 

Figured out how to do a `group_by` from the Namespace model object. It now does a single query for each namespace, rather than multiple queries per Namespace (i.e., one query for each possible content type).

File: galaxy/api/serializers/namespace.py [hunk 1]
```diff
@@ -58,9 +58,15 @@ def get_summary_fields(self, instance):
             'provider': pn.provider.id,
             'provider_name': pn.provider.name.lower()
         } for pn in instance.provider_namespaces.all()]
+
+        content_counts = {}
```
Comment (cutwater) [inline galaxy/api/serializers/namespace.py]: You can use dict expression here:
```
content_counts = {c['content_type__name']: c['count'] for c in instance.content_counts()}
```

File: galaxy/api/views/namespace.py [hunk 2]
```diff
@@ -39,7 +39,7 @@
     'NamespaceProviderNamespacesList',
     'NamespaceProviderNamespacesList',
     'NamespaceContentList',
-    'NamespaceOwnersList',
```
Comment (cutwater) [inline galaxy/api/views/namespace.py]: Why did you remove trailing comma?

File: galaxy/main/models.py [hunk 3]
```diff
@@ -15,9 +15,12 @@
 # You should have received a copy of the Apache License
 # along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
 
+import logging
+
 from django.conf import settings
 from django.core.urlresolvers import reverse
 from django.db import models
+from django.db.models import Count
```
Comment (cutwater) [inline galaxy/main/models.py]: Module `models` already imported on line 22.

File: galaxy/api/views/namespace.py (hunk 2 above)
Comment (chouseknecht) [reply galaxy/api/views/namespace.py]: Fixed

File: galaxy/main/models.py (hunk 3 above)
Comment (chouseknecht) [reply galaxy/main/models.py]: Fixed
